test/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM shellspec/shellspec:kcov
+RUN apk add --no-cache curl jq gawk sed
+COPY ./ /src
```

Comment:
why do you need to copy the `src/` folder?

Reply:
I dont! thanks